packages/react-native-reanimated/android/build.gradle

```
@@ -468,7 +458,7 @@ task prepareReanimatedHeadersForPrefabs(type: Copy) {
 }
 
 tasks.preBuild {
-    dependsOn assertLatestReactNativeWithNewArchitecture, assertMinimalReactNativeVersion
+assertMinimalReactNativeVersion
```

1. Let's add newlines
2. Does it work tho?
```suggestion
    dependsOn assertMinimalReactNativeVersion
```

---

Thanks for pointing it out!